src/main/java/edu/cornell/kfs/krad/service/impl/CuDocumentServiceImpl.java

```diff
@@ -0,0 +1,83 @@
+package edu.cornell.kfs.krad.service.impl;
+
+import com.thoughtworks.xstream.core.BaseException;
+import org.kuali.rice.core.framework.persistence.jta.TransactionalNoValidationExceptionRollback;
+import org.kuali.rice.kew.api.WorkflowDocument;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.krad.maintenance.MaintenanceDocument;
+import org.kuali.kfs.krad.service.MaintainableXMLConversionService;
+import org.kuali.kfs.krad.service.impl.DocumentServiceImpl;
+
+/**
+ * Custom DocumentServiceImpl subclass that performs its own handling
+ * of just-in-time legacy maintenance XML conversion.
+ *
+ * Maintenance documents ordinarily convert the old/new maintenance XML
+ * sections separately if they need it, and are not capable of converting
+ * legacy BO notes XML. To support better conversion of such content,
+ * the default MaintainableXMLConversionService has been switched over
+ * to a no-op implementation, and instead this class will perform
+ * the XML conversion on the entire maintenance XML payload if necessary.
+ * This allows for adding the features without overlaying Rice code.
+ *
+ * There are three get-by-ID service methods that do not save documents
```

Comment:
The paragraph beginning at this line does not accurately reflect the KFS-side DocumentService customization, which appears to be overlaying the DocumentService to make a private method protected (instead of doing full overrides of the 3 get-by-ID methods that Cynergy does). It might be a good idea to update this Javadoc for accuracy.